draft-ietf-tls-exported-authenticator.md

```diff
@@ -291,7 +298,7 @@ The following sections describes APIs that are considered necessary to implement
 The "request" API takes as input:
 
 * certificate_request_context (from 0 to 255 bytes)
-* set of extensions to include (this MUST include signature_algorithms)
+* set of extensions to include (this MUST include signature_algorithms_cert)
```

Comment:
See above.  The original text was correct, by my understanding at least.

Reply:
All this is correct. I had a misconception about the two extensions that has since been corrected.